modular_citadel/code/modules/admin/verbs/debug_vr.dm

```diff
@@ -0,0 +1,51 @@
+/datum/admins/proc/quick_nif()
+	set category = "Fun"
+	set name = "Quick NIF"
+	set desc = "Spawns a NIF into someone in quick-implant mode."
+
+	var/input_NIF
+
+	if(!check_rights(R_ADMIN))
+		return
+
+	var/mob/living/carbon/human/H = input("Pick a mob with a player","Quick NIF") as null|anything in player_list
+
+	if(!H)
+		return
+
+	if(!istype(H))
+		to_chat(usr,"<span class='warning'>That mob type ([H.type]) doesn't support NIFs, sorry.</span>")
+		return
+
+	if(!H.get_organ(BP_HEAD))
+		to_chat(usr,"<span class='warning'>Target is unsuitable.</span>")
+		return
+
+	if(H.nif)
+		to_chat(usr,"<span class='warning'>Target already has a NIF.</span>")
+		return
+
+	if(H.species.flags & NO_SCAN)
+		var/obj/item/device/nif/S = new /obj/item/device/nif/bioadap
+		input_NIF = S.name
+		new /obj/item/device/nif/bioadap(H)
+	else
+		var/list/NIF_types = typesof(/obj/item/device/nif)
+		var/list/NIFs = list()
+
+		for(var/NIF_type in NIF_types)
+			var/obj/item/device/nif/S = new NIF_type
```

Comment:
You're making one completely new NIF of every type just for this single proc. I've also got my own contestant for this coming in with the other NIF changes.

Reply:
Decent catch, tbh, about the new thing.
I just copied my earlier code without re-reading it, just made sure it compiles and works fine.